department-of-veterans-affairs/vets-website
# Pension: add income sources

## Summary

On Step 5 of Financial Information: income,

- Create a new page for the new Add an income source questions, 4 total
- Include multi-entry option for the same questions
- Include supporting copy that provides examples
- Match all copy to designs

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the pension_form_enabled flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run vets-website and vets-api

## How to verify
1. Go to http://localhost:3001/pension/application/527EZ in your browser
2. Select 'Start your application without signing in'
3. Proceed to Chapter 5 by filling out the required fields in previous chapters
4. On the question, "Do you, your spouse, or your dependents receive income?", say 'yes'
5. On the next question, confirm that the page and questions match the designs

## Related issue(s)

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71345)
- [Link to Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done


## Screenshots

| Before | After | After |
| ------- | ----------------|--------------|
|  N/A (new question) |  <img width="299" alt="Screenshot 2023-12-13 at 10 49 13 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/5ac9f6a1-4807-4a0f-9d84-eedd39ad41de"> | <img width="396" alt="Screenshot 2023-12-13 at 10 49 32 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/b891e353-49f3-486a-839e-b0bb63cad5e5"> |


## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71345)

### Quality Assurance & Testing


### Error Handling


### Authentication
